scss/_base_forms.scss

```
@@ -271,7 +271,7 @@
 
   // stylelint-disable-next-line selector-max-type -- base styles can use type selectors
   option:checked:not(:disabled) {
-    background-color: $color-background-active;
+    background-color: rgba($color-background-option, 0.5);
```

Hey, thanks for suggestion, but I'm not sure if we want to do it as a transparent version of the colour, as it should use the colours directly available in Vanilla.

I'm not sure what is causing the issue (will try to debug it), but maybe it's an issue that background is changing but text color is not? So maybe making sure we set text colour would fix it?

```suggestion
    background-color: $color-background-active;
    color: $color-text;
```

---

I've tried setting the text colour but it doesn't work for the dark theme case, since the `$color-text` would be white, similar to the existing `background-color`. 

I wasn't sure if the current background color is the intended color. If so, would changing the text color be sufficient in this case? 

Thanks!

---

Yes, text colour is meant to be white in dark theme.

So I guess the issue is with the `$color-background-active`? 
It uses the value of `$colors--dark-theme--background-active` which is transparent white. Maybe that's an issue. Maybe transparency doesn't work (there is no colour behind), so it turns white. 

With your fix also, even though you seem to be using transparent version of `$color-background-option` it looks the same as other options, so probably just using `$color-background-option` instead of `rgba($color-background-option, 0.5);` would do the same…

I guess to make it work we would need the "active" colour that is not transparent.

These colors are also affecting the multiselect version: https://vanillaframework.io/docs/examples/base/forms/select-multiple

So we need to make sure it works there as well, both in light and dark. 

I don't have a good sugestion right now how to fix this to be honest. Let me think about it.